### ISSUE TITLE
Add support for remote cleanup

### ISSUE THREAD

**Issue by andrelopez** [enupal/backup#26]

**Comment by nitech** [comment]
Yes, this would be nice. Also it would be nice to have a built in scheduler that would schedule backing up and cleaning up old backups, ie: 

Clean up old backups: Yes

Number of hourly backups to keep: x
Number of weekly backups to keep: x
Monthly: x
Yearly: x

**Comment by CreateSean** [comment]
Yes, please set up some way to auto-prune backups.
